Makefile

```diff
@@ -206,3 +206,22 @@ image-bundle/image.list: k0s
 image-bundle/bundle.tar: image-bundle/image.list
 	$(MAKE) -C image-bundle bundle.tar
 
+ifeq ($(OS),Windows_NT)
+detected_OS := Windows
+else
+detected_OS := $(shell uname)
+endif
+
+ifeq ($(detected_OS),Darwin)
+sedopt := -i "" -e
+else
+sedopt := -i -e
+endif
+
+docs/cli: static/gen_manifests.go $(shell find ./cmd/ -type f)
+	mkdir -p docs/cli
```

Comment:
Looks like it still needs `EMBEDDED_BINS_BUILDMODE=none make pkg/assets/zz_generated_offsets_linux.go` run before, otherwise it fails with

```
 k0s git:(fc6be9f4) ✗ make clean docs/cli
docker volume rm k0sbuild.gocache
Error response from daemon: remove k0sbuild.gocache: volume is in use - [392bf56b380912fb75c1b32cb0fe845ef783c24023e0ab19dda5a3ea09027f55]
make: [Makefile:166: clean-gocache] Error 1 (ignored)
rm .k0sbuild.docker-vol.gocache
rm: cannot remove '.k0sbuild.docker-vol.gocache': No such file or directory
make: [Makefile:167: clean-gocache] Error 1 (ignored)
docker rmi k0sbuild.docker-image.k0s -f
Error: No such image: k0sbuild.docker-image.k0s
rm -f .k0sbuild.docker-image.k0s
rm -f pkg/assets/zz_generated_offsets_*.go k0s k0s.exe .bins.*stamp bindata* static/gen_manifests.go
make -C embedded-bins clean
make[1]: Entering directory '/home/ubuntu/projects/k0s/embedded-bins'
for i in .container.*; do \
	if [ -f $i ]; then \
		docker rm $(cat $i) && rm $i; \
	fi; \
done
for i in .docker-image.*; do \
	if [ -f $i ]; then \
		docker rmi -f k0sbuild$(basename $i .stamp) && rm $i;\
	fi; \
done
rm -rf staging .tmp/* *.tmp
make[1]: Leaving directory '/home/ubuntu/projects/k0s/embedded-bins'
make -C image-bundle clean
make[1]: Entering directory '/home/ubuntu/projects/k0s/image-bundle'
rm -f bundle.tar
rm -f .docker-bundler.stamp
make[1]: Leaving directory '/home/ubuntu/projects/k0s/image-bundle'
make -C inttest clean
make[1]: Entering directory '/home/ubuntu/projects/k0s/inttest'
rm -rf bin sonobuoy/*_sonobuoy_*.tar.gz .*.stamp
make[1]: Leaving directory '/home/ubuntu/projects/k0s/inttest'
/home/ubuntu/go/bin/go-bindata -o static/gen_manifests.go -pkg static -prefix static static/...
mkdir -p docs/cli
go run main.go docs markdown
# github.com/k0sproject/k0s/pkg/assets
pkg/assets/stage.go:82:16: assignment mismatch: 2 variables but 1 value
pkg/assets/stage.go:82:19: undefined: BinData
pkg/assets/stage.go:102:28: undefined: BinDataSize
make: *** [Makefile:223: docs/cli] Error 2
```